app/autolinker/client/client.js

```diff
@@ -59,7 +59,7 @@ const renderMessage = (message) => {
 		regexTokens = new RegExp(`(${ (message.tokens || []).map(({ token }) => RegExp.escape(token)) })`, 'g');
 		msgParts = message.html.split(regexTokens);
 	} else {
-		msgParts = [message.html];
+		msgParts = [message.msg];
```

Comment:
```suggestion
		msgParts = [message.html];
```

Reply:
Thank's for feedback @ggazzo, 

I had not realized, sorry. I rollback my changes and added your changes in my commit